src/main/java/models/Game.java

```diff
@@ -7,25 +7,27 @@
 /**
  * Created by michaelhilton on 1/25/16.
  */
-public class Game {
+public abstract class Game {
 
     public java.util.List<Card> deck = new ArrayList<>();
 
     public java.util.List<java.util.List<Card>> cols = new ArrayList<>();
 
-    public int score;
+    public int score = 0;
 
-    public int cardsLeft;
+    public int cardsLeft = 0;
 
     public Game(){
         cols.add(new ArrayList<Card>());
         cols.add(new ArrayList<Card>());
         cols.add(new ArrayList<Card>());
         cols.add(new ArrayList<Card>());
+        this.buildDeck();
```

Comment:
shuffle() wasnt called in the constructor before. It has always been called in the gameGet() method in the Application controller

right now, i have placed the buildDeck() in the Game class constructor because I've made the Game class into an abstract class, and initialized buildDeck() as an abstract method.

This abstract method will be different in differrent child cases and will be called when the Game() constructor is called.

Basically i've placed it there because I didn't want to create another buildDeckSpanish() method. And since we have decided to use the Template inheritence design pattern, this would be the way to go to show that we are using inheritence in our project! :D

Sorry if this posts confuses you as it is quite long!
Shoot me more questions if your still in doubt!